integration/command.go

```diff
@@ -109,19 +109,19 @@ func (c *command) run(t *testing.T) {
 	t.Logf("command: %s\n", c.cmd)
 	cmd := exec.Command("/bin/sh", "-c", c.cmd)
 	output, err := cmd.CombinedOutput()
-	actual := string(output)
-	t.Logf("command returned:\n%s\n", actual)
```

Comment:
I think it's good to print the command before making the test fail. Otherwise we won't know what's the output of the command if it fails (returns with exit code different than 0).

Reply:
This log is not the command that is being executed but the output of the command which will be for sure empty if `cmd.CombinedOutput()` fails. 

Anyway, I will make a separate PR with a couple of changes like this because this logic of running the command and verifying the output is repeated in three different places.